dev-lang/vala/vala-0.36.11.recipe

```diff
@@ -28,26 +28,35 @@ PROVIDES="
 	cmd:valac_${portVersion%.*}$secondaryArchSuffix
 	cmd:vapicheck_${portVersion%.*}$secondaryArchSuffix
 	cmd:vapigen_${portVersion%.*}$secondaryArchSuffix
-	lib:libvala_${portVersion%.*}$secondaryArchSuffix
-	"
-PROVIDES_common="
-	vala${secondaryArchSuffix}_common = $portVersion
+	lib:libvala_${portVersion%.*}$secondaryArchSuffix = 0.0.0 compat >= 0
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
 	glib2${secondaryArchSuffix} >= 2.32.0
 	vala${secondaryArchSuffix}_common == $portVersion
+	lib:libdbus_1$secondaryArchSuffix
 	lib:libffi$secondaryArchSuffix
 	lib:libiconv$secondaryArchSuffix
 	lib:libintl$secondaryArchSuffix
 	"
+PROVIDES_common="
+	vala${secondaryArchSuffix}_common = $portVersion
+	"
 REQUIRES_common="
 	haiku$secondaryArchSuffix
 	"
+PROVIDES_devel="
+	vala${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	devel:libvala_${portVersion%.*}$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
```

Comment:
Not sure here, there is no lib in the develop package, only the *.pc files and headers, would it be safe to remove this line?

Reply:
I don't know.
Does haikuporter complain if you remove that line and run `haikuporter -S vala` ?
If not, then I'd say that you can remove the line.